apache2/msc_multipart.c

```diff
@@ -20,6 +20,9 @@
 #include "msc_util.h"
 #include "msc_parsers.h"
 
+static const char* mime_charset_special = "!#$%&+-^_`{}~";
```

Comment:
In case we allow something else than UTF-8 & ISO-8859-1, these characters look extremely permissive.
The standard allows them obviously, but did you ever encounter such a charset?

Reply:
That's a good question we want to handle the (currently) allowed charset (as you wrote) or we permit anything.

Moreover as I wrote [here](https://github.com/owasp-modsecurity/ModSecurity/pull/3239#issuecomment-2309893202) we should examine the possibility of applying the rules.